astroplan/tests/test_core.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord)
+import astropy.units as u
+from astropy.time import Time
+from astropy.tests.helper import remote_data
```

Comment:
It's covered in the [Test Guidelines](http://astropy.readthedocs.org/en/latest/development/testguide.html#working-with-data-files) in such a way that it's almost advertising for you to borrow the method.